Misc/NEWS.d/next/Library/2022-01-24-23-55-30.bpo-46491.jmIKHo.rst

```diff
@@ -0,0 +1 @@
+Allow Annotated on outside of Final/ClassVar.
```

Comment:
This should make clear that it's about typing. You can use RST like in the docs, e.g. ```:data:`typing.Annotated`.```

Reply:
Thanks, this should be okay now